src/renderer/controllers/torrent-list-controller.js

```diff
@@ -40,6 +40,11 @@ module.exports = class TorrentListController {
 
   // Shows the Create Torrent page with options to seed a given file or folder
   showCreateTorrent (files) {
+    // You can only create torrents from the home screen.
+    if (this.state.location.url() !== 'home') {
+      return dispatch('error', 'Please go back to the torrent list before creating a new torrent.')
```

Comment:
Just tried it --- I *can* open Prefs from within Create Torrent. (I just can't open another Prefs from within Prefs, or another Create Torrent from within either of those.)